File: src/main/kotlin/hu/hirschbeckd/topicer/config/properties/KafkaProperties.java
package hu.hirschbeckd.topicer.config.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Data
@Component
@ConfigurationProperties(prefix = "topical.kafka")
public class KafkaProperties {

    private String serverUrls;
}

File: src/main/kotlin/hu/hirschbeckd/topicer/config/properties/KafkaProperties.kt
package hu.hirschbeckd.topicer.config.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding


@ConstructorBinding
@ConfigurationProperties(prefix = "topical.kafka")
data class KafkaProperties(
        val serverUrls: String
)
